src/unitxt/llm_as_judge.py

```diff
@@ -8,43 +8,16 @@
 from .error_utils import UnitxtError
 from .inference import (
     InferenceEngine,
-    OptionSelectingByLogProbsInferenceEngine,
```

Comment:
Although this imports are unused, they are here in order to let users import anything from unitxt.llmasjudge. Lets re-add them.